gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/inspection/inspect.ts

```diff
@@ -0,0 +1,103 @@
+import { Result } from '@chainlink/gauntlet-core'
+import { SolanaCommand, TransactionResponse } from '@chainlink/gauntlet-solana'
+import { PublicKey } from '@solana/web3.js'
+import { CONTRACT_LIST, getContract } from '../../../../lib/contracts'
+import { getRDD } from '../../../../lib/rdd'
+import {
+  inspect,
+  Inspection,
+  makeInspection,
+  toComparableNumber,
+  toComparablePubKey,
+} from '../../../../core/inspection'
+
+type Input = {
+  description: string
+  decimals: string | number
+  minAnswer: string | number
+  maxAnswer: string | number
+  transmitters: string[]
+  billingAccessController: string
+  requesterAccessController: string
+  link: string
+}
+
+export default class OCR2Inspect extends SolanaCommand {
+  static id = 'ocr2:inspect'
+  static category = CONTRACT_LIST.OCR_2
+
+  makeInput = (userInput): Input => {
+    if (userInput) return userInput as Input
+    const rdd = getRDD(this.flags.rdd)
+    const info = rdd.contracts[this.flags.state]
+    const aggregatorOperators: string[] = info.oracles.map((o) => o.operator)
+    const transmitters = aggregatorOperators.map((operator) => rdd.operators[operator].nodeAddress[0])
+    const billingAccessController =
+      this.flags.billingAccessController || process.env.BILLING_ADMIN_ACCESS_CONTROLLER_ADDRESS
+    const requesterAccessController =
+      this.flags.requesterAccessController || process.env.REQUESTER_ADMIN_ACCESS_CONTROLLER_ADDRESS
+    const link = this.flags.link || process.env.LINK_ADDRESS
+    return {
+      description: info.name,
+      decimals: info.decimals,
+      minAnswer: info.minSubmissionValue,
+      maxAnswer: info.maxSubmissionValue,
+      transmitters,
+      billingAccessController,
+      requesterAccessController,
+      link,
+    }
```

Comment:
This is a subset of the config stored onchain, which is also present in the RDD, right? 
I'm asking because there is a lot more stuff onchain that we could be checking on. Eg. https://github.com/smartcontractkit/offchain-reporting/blob/d6773dfde7b48d814d7d8dec1a0e85ad8d6eb01a/lib/offchainreporting2/internal/config/offchainreporting2_offchain_config.proto or https://github.com/smartcontractkit/offchain-reporting/blob/d6773dfde7b48d814d7d8dec1a0e85ad8d6eb01a/lib/offchainreporting2/reportingplugin/median/offchainreporting2_median_config.proto

Reply:
Yes you're right, I'll add the offchain config validation